kmk/modules/combos.py

```diff
@@ -127,7 +139,9 @@ def on_press(self, keyboard: KMKKeyboard, key: Key, int_coord: Optional[int]):
         for combo in self.combos:
             if combo._state != _ComboState.MATCHING:
                 continue
-            if combo.matches(key):
+            if isinstance(combo, (Chord, Sequence)) and combo.matches(key):
+                continue
+            elif isinstance(combo, Coord) and combo.coord_matches(int_coord):
                 continue
```

Comment:
I'm not a big fan of that pattern. We should pass both `key` and `int_coord` into `combo.matches()`, and decide there which one to use.

Reply:
yeah thats actually a good idea, I wasn't sure if my approach is a good practice, will make the changes